autoPyTorch/pipeline/tabular_classification.py

```diff
@@ -91,12 +91,7 @@ def _predict_proba(self, X: np.ndarray) -> np.ndarray:
         loader = self.named_steps['data_loader'].get_loader(X=X)
         pred = self.named_steps['network'].predict(loader)
         if isinstance(self.dataset_properties['output_shape'], int):
-            proba = pred[:, :self.dataset_properties['output_shape']]
-            normalizer = proba.sum(axis=1)[:, np.newaxis]
-            normalizer[normalizer == 0.0] = 1.0
-            proba /= normalizer
-
-            return proba
+          return pred
```

Comment:
Could you explain the reason behind this change?
Btw, the indent level is different in the new change.

Reply:
I think this change was suggested by Ravin. The idea was that the predictions did not need to be normalized, @ravinkohli can explain it better. I think at a certain time the code below was creating an error, but not anymore.